Troubles with dependencies during installation

I just started with fresh Ubuntu 20.04, and decided to try nvim instead of gvim I used to work in. Now looking for GUI for it.

Having Go already installed, I did

```
go install github.com/hismailbulut/neoray/src@latest
```

And got the following fatal errors, one after another fixing them one by one: 

```
X11/Xlib.h: No such file or directory
X11/Xcursor/Xcursor.h: No such file or directory
X11/extensions/Xrandr.h: No such file or directory
X11/extensions/Xinerama.h: No such file or directory
X11/extensions/XInput2.h: No such file or directory
fatal error: GL/glx.h: No such file or directory
Package gtk+-3.0 was not found in the pkg-config search path.
```

These are fixed by 
```
sudo apt install libx11-dev libxcursor-dev libxrandr-dev libxinerama-dev libxi-dev libgl1-mesa-dev build-essential libgtk-3-dev
```

It would be nice to mention this somewhere in documentation about installation.

Now I'm stuck with
```
# github.com/go-gl/glfw/v3.3/glfw
/usr/bin/ld: cannot find -lXxf86vm
```

Not sure what that means.

--- a/README.md
+++ b/README.md
@@ -5,14 +5,28 @@ golang using glfw and opengl bindings. Neoray is easy to use and binary size
 is small. Supports most of the neovim features. Uses small amount of ram and
 leaves no footprints on your computer.
 
-You can install neoray with go install command.
+## Installation
+
+You can install neoray with `go install` command:
 ```
 go install github.com/hismailbulut/neoray/src@latest
 ```
+
+NOTE: to successfully build it with go you will need to have
+some dependencies. On Ubuntu, install following packages:
+
+```
+sudo apt install libx11-dev libxcursor-dev libxrandr-dev libxinerama-dev libxi-dev libgl1-mesa-dev build-essential libgtk-3-dev xorg-dev
+```
+
 Also you can download prebuild binaries from releases page.
 
+
+## Configuration
+
 Neoray doesn't need any additional configuration, but you can customize it in
-your init.vim.
+your `init.vim`. But for `neoray_*` settings to apply it is not enought to
+`:source` it, you'll need to restart neoray.
 
 These are some options you can specify for now:
 

